Sources/Client/TezosClient.swift

```diff
@@ -229,6 +229,62 @@ public class TezosClient {
                                         completion: completion)
   }
 
+  /**
+   * Retrieve ballots cast so far during a voting period.
+   */
+  public func getBallotsList(completion: @escaping ([[String: Any]]?, Error?) -> Void) {
```

Comment:
Why not use promises https://github.com/mxcl/PromiseKit ?

Can an actual parsed type be returned instead of an array of dictionaries?  Perhaps a type alias?